File: Bahasa c/PRAK305-2210817220030-Valencia Angelin Nata.c
#include <stdio.h>
int main(){
int bd,m,j,d,h;
scanf("%d", &bd);
h= (bd%(86400*30))/86400;
j= (bd%86400)/3600;
m= (bd%3600)/60;
d= (bd%60);
if (h==0){
    printf ("%02d:%02d:%02d\n\n",j,m,d);
    }
else{
    printf("%d hari %02d:%02d:%02d\n\n",h,j,m,d);
    }
}

File: Bahasa python/PRAK305-2210817220030-Valencia Angelin Nata.py
bd= int (input())
h= (bd%(86400*30))/86400
j= (bd%86400)/3600
m= (bd%3600)/60
d= (bd%60)
if (h>=1):
    print ("%d hari %02d:%02d:%02d\n"%(h,j,m,d))
else:
    print("%02d:%02d:%02d\n"%(j,m,d))